apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/src/test/resources/GenerateEntitiesFromDslMojoIT/expected/FissProcCode.java

```diff
@@ -142,7 +144,8 @@ public void setLastUpdated(Instant lastUpdated) {
   }
 
   /**
-   * PK class for the FissProcCodes table */
+   * PK class for the FissProcCodes table
+   */
```

Comment:
is this from a change to the formatter version or something?

Reply:
Something in 1.12 onward of javapoet is causing comments like these to be parsed differently. Since these are test files in one project, and the issue was fixed by just making a multiline comment, Id say we can look further into this in the future if something comes up.